dev/mock_tbl.R

```diff
@@ -29,7 +29,10 @@ tfrmt_spec  <- tfrmt(
                    frmt_combine(
                      "{count} {percent}",
                      count = frmt("XXX"),
-                     percent = frmt("(XXX.X%)")
+                     percent = frmt_when("==100"~ frmt(""),
```

Comment:
how is frmt("") different from ""?

Reply:
It isn't really. But I think it is reasonable that people will use both options so I want to make sure both work